### PR TITLE
[stable/spinnaker] Making permissions to write for user/group as a variable

### PR DESCRIPTION
**Pull request by ElizabethSvit**
#### What this PR does / why we need it:

One might need to be able to pass a custom id for the permissions to write. PR does provide an additional field in values.yaml for the rights (1000 by default).

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by rochaporto** [comment]
Also seems to work nicely, thanks @ElizabethSvit

File: stable/spinnaker/templates/statefulsets/halyard.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ spec:
       serviceAccountName: {{ template "spinnaker.fullname" . }}-halyard
       {{- end }}
       securityContext:
-        runAsUser: 1000
-        fsGroup: 1000
+        runAsUser: {{ .Values.securityContext.userRights }}
```
Comment (paulczar) [inline stable/spinnaker/templates/statefulsets/halyard.yaml]: please set these variable names to be `runAsUser` and `fsGroup` to match the `securityContext` keys.

Comment (ElizabethSvit) [reply stable/spinnaker/templates/statefulsets/halyard.yaml]: done

**Pull request merged**